src/components/username-form-item.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Input } from 'antd';
+import { useTranslation } from 'react-i18next';
+import { minUsernameLength } from '../domain/validation-constants.js';
+
+const FormItem = Form.Item;
+
+function UsernameFormItem({ name, usernamesInUse, ...formItemProps }) {
```

Comment:
We would need to pass in the initial value here for the case of the account tab, where the username needs to be prefilled

Reply:
Thanks for spotting that, it's passed in there now.